docs/markdown/snippets/wrap_patch.md

```diff
@@ -4,3 +4,9 @@ It is now possible to use the `patch_filename` and `source_filename` value in a
 `.wrap` file without `*_url` to specify a local source / patch file. All local
 files must be located in the `subprojects/packagefiles` directory. The `*_hash`
 entries are optional with this setup.
+
+## Local wrap patch directory
+
+Wrap files can now specify `patch_directory` instead of `patch_filename` in the
+case overlay files are local. The patch directory must be placed in
+`subprojects/packagefiles` directory.
```

Comment:
"Overlay" is not an immediately obvious term. This should explain briefly what it does (e.g. "copies files in this dir unaltered over the source dir").

Reply:
done.

Reply:
Note that the doc already uses the term 'overlay' for this, see https://mesonbuild.com/Wrap-dependency-system-manual.html#specific-to-wrapfile. But I agree it's better to clarify in the release snippet too.